mopidy/http/actor.py

```diff
@@ -45,7 +47,14 @@ def __init__(self, config, core):
 
         try:
             logger.debug("Starting HTTP server")
-            sockets = tornado.netutil.bind_sockets(self.port, tornado_hostname)
+            try:
+                fdnames = os.environ.get("LISTEN_FDNAMES", "").split(":")
+                fd = fdnames.index("http") + 3 # got a socket from systemd
```

Comment:
Where does the 3 come from?

Reply:
It's a systemd magic number. The file descriptor for the first socket it hands over is 3, the next one is 4, etc. It's described here https://www.freedesktop.org/software/systemd/man/sd_listen_fds.html#

Reply:
That should be detailed in a comment in the code, so it’s always easily available to maintainers.